.env

```diff
@@ -0,0 +1,3 @@
+NEXT_PUBLIC_TITLE_URL='famf.app'
+NEXT_PUBLIC_MAX_IMAGE_UPLOAD_SIZE_MB=2
+NEXT_PUBLIC_MAX_CSV_UPLOAD_SIZE_MB=2
```

Comment:
Do these need to be pre-pended with `NEXT_` in order for nextjs to pick them up?

Reply:
Yes, see [Environment Variables](https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#bundling-environment-variables-for-the-browser)

> By default, environment variables are only available on the server. To expose an environment variable to the browser, it must be prefixed with NEXT_PUBLIC_.

Reply:
Some [examples](https://github.com/vercel/next.js/tree/canary/examples/environment-variables)